packages/slice-machine/src/domain/customType.ts

```diff
@@ -425,6 +453,64 @@ export function reorderField(args: ReorderFieldArgs): CustomType {
   return newCustomType;
 }
 
+// if the UID is not existing in any section, it should be added to the main section
+export function addUIDField(label: string, customType: CustomType): CustomType {
+  const sectionId = getMainSectionId(customType);
+  if (typeof sectionId === "undefined") {
+    return customType;
+  }
+  const newFieldId = "uid";
+  const newField: UID = {
+    config: {
```

Comment:
⚠️ #issue

Potential issue, did you discuss the placeholder? I don't see it here at all, but how would a dev configure the placeholder visible for the editor?

Reply:
That's good observation, current design doesn't provide any option to configure placeholder - will ask Benjamin!

Reply:
talked with Benjamin, he is ok with not adding placeholder here
also talk to Come, he is also ok with not having option to configure placeholder, but asked to default it to ""Enter a unique identifier"